.changeset/silly-planets-wait.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/tree": minor
+---
+
+Add debug visualizers for NodeJS and browsers for TreeNodes
+
+TreeNodes now have custom debug visualizers which should improve the debug experience in NodeJS and in browsers if custom formatters are enabled in the developer tools.
```

Comment:
```suggestion
TreeNodes now have custom debug visualizers to improve the debug experience in NodeJS and in browsers. Note that custom formatters must be enabled in the developer tools for these visualizers to be used.
```

In this case are the devtools the browser devtools or the Fluid devtools?

Reply:
browser.

```suggestion
TreeNodes now have custom debug visualizers to improve the debug experience in NodeJS and in browsers. Note that custom formatters must be enabled in the browser developer tools for that visualizer to be used.
```